x-pack/plugins/security_solution/public/common/hooks/eql/helpers.ts

```diff
@@ -1,183 +0,0 @@
- * @param relativeNow The timestamp we are using to calculate how far from 'now' event occurred
- */
-export const calculateBucketForDay = (eventTimestamp: number, relativeNow: number): number => {
-  const diff = Math.abs(relativeNow - eventTimestamp);
-  const minutes = Math.floor(diff / 60000);
-  return Math.ceil(minutes / 60);
-};
-
-/**
- * Formats the response for the UI inspect modal
- * @param response The query search response
- * @param indices The indices the query searched
- * TODO: Update eql search strategy to return index in it's meta
- * params info, currently not being returned, but expected for
- * inspect modal display
- */
-export const formatInspect = (
-  response: EqlSearchStrategyResponse<EqlSearchResponse<Source>>,
-  indices: string[]
-): InspectResponse => {
-  const body = response.rawResponse.meta.request.params.body;
-  const bodyParse: Record<string, unknown> | undefined =
-    typeof body === 'string' ? JSON.parse(body) : body;
-  return {
-    dsl: [
-      JSON.stringify(
-        { ...response.rawResponse.meta.request.params, index: indices, body: bodyParse },
-        null,
-        2
-      ),
-    ],
-    response: [JSON.stringify(response.rawResponse.body, null, 2)],
-  };
-};
-
-/**
- * Gets the events out of the response based on type of query
- * @param isSequence Is the eql query a sequence query
- * @param response The query search response
- */
-export const getEventsToBucket = (
-  isSequence: boolean,
-  response: EqlSearchStrategyResponse<EqlSearchResponse<Source>>
-): Array<BaseHit<Source>> => {
-  const hits = response.rawResponse.body.hits ?? [];
-  if (isSequence) {
-    return (
-      hits.sequences?.map((seq) => {
-        return seq.events[seq.events.length - 1];
-      }) ?? []
-    );
-  } else {
-    return hits.events ?? [];
-  }
-};
-
-/**
- * Eql does not support aggregations, this is an in-memory
- * hand-spun aggregation for the events to give the user a visual
- * representation of their query results
- * @param response The query search response
- * @param range User chosen timeframe (last hour, day)
- * @param to Based on range chosen
- * @param refetch Callback used in inspect button, ref just passed through
- * @param indices Indices searched by query
- * @param isSequence Is the eql query a sequence query
- */
-export const getEqlAggsData = (
-  response: EqlSearchStrategyResponse<EqlSearchResponse<Source>>,
-  range: Unit,
-  to: string,
-  refetch: inputsModel.Refetch,
-  indices: string[],
-  isSequence: boolean
-): EqlPreviewResponse => {
-  const { dsl, response: inspectResponse } = formatInspect(response, indices);
-  const relativeNow = Date.parse(to);
-  const accumulator = getInterval(range, relativeNow);
-  const events = getEventsToBucket(isSequence, response);
-  const totalCount = response.rawResponse.body.hits.total.value;
-
-  const buckets = events.reduce<EqlAggBuckets>((acc, hit) => {
-    const timestamp = hit._source['@timestamp'];
```

Comment:
we had something naughty going on here, but these functions were unused anyways so ✂️